app/views/admin/questions/_form.html.erb

```diff
@@ -31,7 +31,7 @@
         <br>
       <% end %>
     <% else %>
-      <%= f.select :question_type, Question::QUESTION_TYPES - Question::LIMITED_QUESTION_TYPES, { selected: question.question_type, include_blank: true }, { class: "usa-select" } %>
+      <%= f.select :question_type, (Question::QUESTION_TYPES - Question::LIMITED_QUESTION_TYPES + [question.question_type]).uniq, { selected: question.question_type, include_blank: true }, { class: "usa-select" } %>
```

Comment:
some questions aren't shown to users, but are available in templates.

this ensures these questions aren't lost when trying to update.